src/Module/Api/Mastodon/FollowRequests.php

```diff
@@ -34,26 +69,32 @@ public static function rawContent(array $parameters = [])
 		$limit = intval($_GET['limit'] ?? 40);
 
 		if (isset($since_id) && isset($max_id)) {
-			$condition = ['`uid` = ? AND NOT `self` AND `pending` AND `id` > ? AND `id` < ?', self::$current_user_id, $since_id, $max_id];
+			$condition = ['`uid` = ? AND NOT `ignore` AND `id` > ? AND `id` < ?', self::$current_user_id, $since_id, $max_id];
 		} elseif (isset($since_id)) {
-			$condition = ['`uid` = ? AND NOT `self` AND `pending` AND `id` > ?', self::$current_user_id, $since_id];
+			$condition = ['`uid` = ? AND NOT `ignore` AND `id` > ?', self::$current_user_id, $since_id];
 		} elseif (isset($max_id)) {
-			$condition = ['`uid` = ? AND NOT `self` AND `pending` AND `id` < ?', self::$current_user_id, $max_id];
+			$condition = ['`uid` = ? AND NOT `ignore` AND `id` < ?', self::$current_user_id, $max_id];
 		} else {
-			$condition = ['`uid` = ? AND NOT `self` AND `pending`', self::$current_user_id];
+			$condition = ['`uid` = ? AND NOT `ignore`', self::$current_user_id];
 		}
 
-		$count = DBA::count('contact', $condition);
+		$count = DBA::count('intro', $condition);
 
-		$contacts = Contact::selectToArray(
+		$intros = DBA::selectToArray(
+			'intro',
 			[],
 			$condition,
 			['order' => ['id' => 'DESC'], 'limit' => $limit]
 		);
 
 		$return = [];
-		foreach ($contacts as $contact) {
-			$account = Account::createFromContact($contact);
+		foreach ($intros as $intro) {
+			$contact = Contact::getById($intro['contact-id']);
+			$apcontact = APContact::getByURL($contact['url'], false);
```

Comment:
You should only use the `false` value here, when you are 100% sure that the APContact is already existing. Is this the case here?

Reply:
It's a retrieval call, not an update one. The absence of the `apcontact` row even if it was supposed to exist doesn't have that big of an impact.